manifests/02-crd-tuned.yaml

```diff
@@ -59,78 +69,66 @@ spec:
+                        profile 'Profile' needs and can only be applied by creating
+                        a MachineConfig. This involves finding all MachineConfigPools
+                        with machineConfigSelector matching the MachineConfigLabels
+                        and setting the profile 'Profile' on all nodes that match
+                        the MachineConfigPools' nodeSelectors.
+                      type: object
                       additionalProperties:
                         type: string
-                      description: MachineConfigLabels specifies the labels for a MachineConfig.
-                        The MachineConfig is created automatically to apply additional
-                        host settings (e.g. kernel boot parameters) profile 'Profile'
-                        needs and can only be applied by creating a MachineConfig. This
-                        involves finding all MachineConfigPools with machineConfigSelector
-                        matching the MachineConfigLabels and setting the profile 'Profile'
-                        on all nodes that match the MachineConfigPools' nodeSelectors.
-                      type: object
                     match:
-                      description: Rules governing application of a tuned profile connected
-                        by logical OR operator.
+                      description: Rules governing application of a tuned profile
+                        connected by logical OR operator.
+                      type: array
                       items:
                         description: Rules governing application of a tuned profile.
+                        type: object
+                        required:
+                        - label
                         properties:
                           label:
                             description: Node or Pod label name.
                             type: string
                           match:
-                            description: 'Additional rules governing application of the tuned profile connected by logical AND operator.'
+                            description: Additional rules governing application of
+                              the tuned profile connected by logical AND operator.
                             type: array
-                            items:
-                              # Recursively nested "match" sections are dropped without the following.
-                              x-kubernetes-preserve-unknown-fields: true
-                              # OpenAPI v3 validation schemas containing $ref are not permitted => cannot do recursive validation in 1.14
-                              # https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#validation
-                              type: object
+                            items: { type: object }
+                            x-kubernetes-preserve-unknown-fields: true
```

Comment:
this looks wrong. Put `x-kubernetes-preserve-unknown-fields: true` into the items schema.

Reply:
I bet kube-apiserver prunes the items schema away right now, but leaves the `x-kubernetes-preserve-unknown-fields`, leading oc to fall over. This is a bug. But this bug should never happen if you follow ^^.